tutorials/lesson1-booleans/booleans.qnt

```diff
@@ -1,42 +1,34 @@
-/**
- This lesson teaches you the basics of operations over Booleans.
- If you have programming experience, you know all of these operators.
- So it should not take you long to finish this lesson.
-
- If you do not know how to start the lesson, do the following:
-
-  - Open Visual Studio Code.
-  - Install the Quint and Codetour extensions.
-  - Choose the tour "Lesson 1 - Booleans".
+/*
+ If you do not know how to start this lesson, see:
+ https://github.com/informalsystems/quint/tree/main/tutorials
  */
 module booleansLesson {    
-    /// false is a built-in constant
-    pure val myFalse = false
 
-    // exercise: evaluate false in REPL
+    // false is a built-in constant
+    pure val myFalse = false
 
-    /// true is a built-in constant too
+    // true is a built-in constant too
```

Comment:
afaiu, Quint doc comments are setoff by 3 `/`. So i'm unclear on the reason for this change.

Reply:
These are not doc documents, but simple one liners